lte/cloud/go/services/lte/lte/main.go

```diff
@@ -40,6 +45,17 @@ func main() {
 
 	builder_protos.RegisterMconfigBuilderServer(srv.GrpcServer, servicers.NewBuilderServicer())
 	provider_protos.RegisterStreamProviderServer(srv.GrpcServer, servicers.NewProviderServicer())
+	state_protos.RegisterIndexerServer(srv.GrpcServer, servicers.NewIndexerServicer())
+	// Init storage
+	db, err := sqorc.Open(storage.SQLDriver, storage.DatabaseSource)
```

Comment:
Note for another time: we should probably change storage.DatabaseSource to not have a default. I think defaulting to in-memory for production code is a risky pattern -- we should just be panic/fatal-ing when that env var isn't set

If you agree go ahead and make a task in the service mesh followups epic. Probably a good bootcamp task

Reply:
Filed the issue